source/_patterns/04-pages/homepage/homepage~en.yaml

```diff
@@ -12,30 +12,37 @@ body:
           - content:
               href: '#'
               heading:
+                level: 4
```

Comment:
Means the heading levels on the page is 1, 2, 4. To style these as if there were a `<h4>`, use the `h4-typography` mixin.